evap/staff/views.py

```diff
@@ -1068,7 +1068,7 @@ def get_evaluation_and_contributor_textanswer_sections(evaluation, filter_textan
         for questionnaire in contribution.questionnaires.all():
             text_results = []
 
-            for question in questionnaire.text_questions:
+            for question in questionnaire.questions.all():
```

Comment:
(Not your change, but I think we should prevent O(N) queries here - probably something for another issue).

Reply:
That should be simple to fix with something like `contribution.questionnaires.prefetch_related('questions')`. The `TextAnswer.objects.filter` below is still O(N) of course.

To not increase the number of queries (right now the PR adds queries for heading results and rating results that don't allow textanswers), you could add a `if not question.allows_textanswers): continue`.